docs/conf.py

```diff
@@ -0,0 +1,77 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'lsst-panel'
+copyright = '2019, Quansight'
+author = 'Quansight'
+
+#need to add path so that function will work that will handle the case where sphinx will
+#ignore certain strings in the code.
+sys.path.insert(0, os.path.abspath('../../nbsite/'))
```

Comment:
I'm assuming this line can be taken out once nbsite is installed into the environment.

Reply:
yes, I took it out.  Thanks!